registrations/migrations/0017_auto_20230810_1303.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.2.20 on 2023-08-10 10:03
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("registrations", "0016_multilingual_confirmation_message_and_instructions"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="signup",
+            name="cancellation_code",
+        ),
+        migrations.AddField(
+            model_name="signup",
+            name="created_at",
+            field=models.DateTimeField(
+                auto_now_add=True, null=True, verbose_name="Created at"
+            ),
+        ),
+        migrations.AddField(
+            model_name="signup",
+            name="created_by",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="signup_created_by",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name="signup",
+            name="last_modified_at",
+            field=models.DateTimeField(
+                auto_now=True, null=True, verbose_name="Modified at"
+            ),
+        ),
+        migrations.AddField(
+            model_name="signup",
+            name="last_modified_by",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="signup_last_modified_by",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="registration",
+            name="created_at",
+            field=models.DateTimeField(
+                auto_now_add=True, null=True, verbose_name="Created at"
+            ),
+        ),
+    ]
```

Comment:
It would be nicer for the future if the migration had some human-readable name.

Reply:
Migration name has been changed in the most recent commit.